scripts/pylib/twister/twisterlib.py

```diff
@@ -723,6 +723,7 @@ def handle(self):
             serial_device = hardware.serial
 
         logger.debug("Using serial device {}".format(serial_device))
+        logger.debug("Using serial baud {}".format(hardware.serial_baud))
```

Comment:
```suggestion
        logger.debug("Using serial device {} @ {}".format(serial_device, hardware.serial_baud))
```

Perhaps combine the two to reduce clutter?